### ISSUE TITLE
Add Tidelift to a secondary "you are seeking funding" list/resource

### ISSUE THREAD

**Issue by brainwane** [psf/project-funding-wg#4]
Some open source maintainers don't yet know [about funding available from Tidelift](https://tidelift.com/lifter/packages-with-income) for the packages they already maintain. I think we should add it to the funders list, but I am currently a consultant for Tidelift (just a few hours per year) so I'm a bit biased and I want others' opinions first. Thoughts?

@ericholscher @marlenezw @di @xmunoz @jackiekazil?

**Comment by di** [comment]
My understanding is that the `FUNDERS.md` list is for organizations that this WG either has requested funding from, or may someday request funding from. This list would be a good place for us to start when attempting to fund a specific project, or a good place to occasionally go through and check for new funding opportunities.

It seems like Tidelift's sponsorship is more 1:1 with the given package maintainer and thus wouldn't require the intervention or assistance of the WG.

I agree this is a valuable resource though. Maybe there's a better place to put it? Maybe we need a guide for "so you have a project that wants funding..." where we could suggest this as an alternative to engaging w/ the WG, at least for small amounts of funding?

**Comment by xmunoz** [comment]
I agree with @di on this, though I do think it's worth moving the conversation to another channel. If there were a way PSF could help package developer and maintainers make money for their packages, that would be awesome.

**Comment by xmunoz** [comment]
Maybe Tidelift could be part of #11?

**Comment by brainwane** [comment]
I think it's fine for us to add Tidelift to some kind of "so you want funding" resource, and I like @di's suggestion.

**Comment by brainwane** [comment]
OK, it sounds like the resolution is that Tidelift and similar mechanisms should go in some secondary list/resource.
